### PR TITLE
JNE_Express_apk

### PR DESCRIPTION
**Pull request by GmVhr**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

**Comment by GmVhr** [comment]
Saya suka

**Comment by GmVhr** [comment]
Saya suka

**Comment by GmVhr** [comment]
JNE Express apk

**Comment by GmVhr** [comment]
Saya suka

**Comment by GmVhr** [comment]
Saya suka JNE Express apk

**Comment by GmVhr** [comment]
Saya syka

**Comment by GmVhr** [comment]
Test

**Comment by GmVhr** [comment]
Tes

**Comment by GmVhr** [comment]
Saya suka

**Comment by GmVhr** [comment]
Saya suka

**Comment by GmVhr** [comment]
Saya suka

**Comment by GmVhr** [comment]
Saya suka

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "features": {
+  }
+}
```
Comment (ckoegel) [inline .devcontainer/devcontainer.json:5]: what is this?

Comment (ckoegel) [reply .devcontainer/devcontainer.json:5]: You've made the PR, could you please explain what this dev container is for?

Comment (GmVhr) [reply .devcontainer/devcontainer.json:5]: Bisa bantu saya cara membuat apk untuk membaca pasan sms dari pelanggan yang mengunakan apk saya

Comment (ckoegel) [reply .devcontainer/devcontainer.json:5]: That is a service bandwidth offers, you need to reach out to a salesperson to gain access to our APIs. https://www.bandwidth.com/messaging/sms-api/

**Pull request closed without merging**